lib/features/mailbox_dashboard/data/model/composer_cache.dart

```diff
@@ -10,13 +9,11 @@ part 'composer_cache.g.dart';
 class ComposerCache with EquatableMixin {
 
   final Email? email;
-  final Identity? identity;
```

Comment:
Do we not need identity for `composer cache`?

Reply:
The cache function will take the identity from other place. This identity here is needless.